sentry-ruby/spec/spec_helper.rb

```diff
@@ -44,6 +44,12 @@
   config.before(:each, rack: true) do
     skip("skip rack related tests") unless defined?(Rack)
   end
+
+  RSpec::Matchers.define :have_recorded_lost_event do |reason, type|
```

Comment:
@st0012 not an rspec pro so not sure if this is the right way to do this? let me know :)

Reply:
I think it looks good and I've also tested it locally 👍